tests/inputs/examples.py

```diff
@@ -84,6 +84,7 @@
0,&quot;ClimateChange&quot;:{&quot;Value&quot;:0.0751787094942408,&quot;Unit&quot;:&quot;kg&quot;},&quot;ClimateChangePercentage&quot;:100}],&quot;JoiningAndFinishingProcesses&quot;:[],&quot;PhaseSummary&quot;:{&quot;Phase&quot;:&quot;Processes&quot;,&quot;EmbodiedEnergy&quot;:{&quot;Value&quot;:17.9476406266453,&quot;Unit&quot;:&quot;MJ&quot;},&quot;EmbodiedEnergyPercentage&quot;:0.112321145492071,&quot;ClimateChange&quot;:{&quot;Value&quot;:0.0751787094942408,&quot;Unit&quot;:&quot;kg&quot;},&quot;ClimateChangePercentage&quot;:0.00348800188134608}},&quot;TransportSummary&quot;:{&quot;Summary&quot;:[{&quot;StageName&quot;:&quot;Aircraft, short haul, belly-freight&quot;,&quot;RecordReference&quot;:{&quot;ReferenceType&quot;:&quot;MiRecordGuid&quot;,&quot;ReferenceValue&quot;:&quot;b916ed6b-5e06-4343-9131-d4d562e2d12b&quot;,&quot;Id&quot;:&quot;P6&quot;},&quot;Distance&quot;:{&quot;Value&quot;:200,&quot;Unit&quot;:&quot;km&quot;},&quot;EmbodiedEnergy&quot;:{&quot;Value&quot;:4.72415244615649,&quot;Unit&quot;:&quot;MJ&quot;},&quot;EmbodiedEnergyPercentage&quot;:100,&quot;ClimateChange&quot;:{&quot;Value&quot;:0.333123822320444,&quot;Unit&quot;:&quot;kg&quot;},&quot;ClimateChangePercentage&quot;:100}],&quot;PartSummary&quot;:[{&quot;ParentPartName&quot;:&quot;&quot;,&quot;PartName&quot;:&quot;&quot;,&quot;Category&quot;:&quot;Manufacturing&quot;,&quot;TransportTypes&quot;:[&quot;Aircraft, short haul, belly-freight&quot;],&quot;Distance&quot;:{&quot;Value&quot;:200,&quot;Unit&quot;:&quot;km&quot;},&quot;EmbodiedEnergy&quot;:{&quot;Value&quot;:4.72415244615649,&quot;Unit&quot;:&quot;MJ&quot;},&quot;EmbodiedEnergyPercentage&quot;:100,&quot;ClimateChange&quot;:{&quot;Value&quot;:0.333123822320444,&quot;Unit&quot;:&quot;kg&quot;},&quot;ClimateChangePercentage&quot;:100}],&quot;CategorySummary&quot;:[{&quot;Category&quot;:&quot;Distribution&quot;,&quot;Distance&quot;:{&quot;Value&quot;:0,&quot;Unit&quot;:&quot;km&quot;},&quot;EmbodiedEnergy&quot;:{&quot;Value&quot;:0,&quot;Unit&quot;:&quot;MJ&quot;},&quot;EmbodiedEnergyPercentage&quot;:0,&quot;ClimateChange&quot;:{&quot;Value&quot;:0,&quot;Unit&quot;:&quot;kg&quot;},&quot;ClimateChangePercentage&quot;:0},{&quot;Category&quot;:&quot;Manufacturing&quot;,&quot;Distance&quot;:{&quot;Value&quot;:200,&quot;Unit&quot;:&quot;km&quot;},&quot;EmbodiedEnergy&quot;:{&quot;Value&quot;:4.72415244615649,&quot;Unit&quot;:&quot;MJ&quot;},&quot;EmbodiedEnergyPercentage&quot;:100,&quot;ClimateChange&quot;:{&quot;Value&quot;:0.333123822320444,&quot;Unit&quot;:&quot;kg&quot;},&quot;ClimateChangePercentage&quot;:100}],&quot;PhaseSummary&quot;:{&quot;Phase&quot;:&quot;Transport&quot;,&quot;EmbodiedEnergy&quot;:{&quot;Value&quot;:4.72415244615649,&quot;Unit&quot;:&quot;MJ&quot;},&quot;EmbodiedEnergyPercentage&quot;:0.0295650122079945,&quot;ClimateChange&quot;:{&quot;Value&quot;:0.333123822320444,&quot;Unit&quot;:&quot;kg&quot;},&quot;ClimateChangePercentage&quot;:0.0154556592789601}},&quot;LogMessages&quot;:[]}""",  # noqa: E501
```

Comment:
We probably want to rethink how this works. It's useful having a blob to assert from, but it's a pain to set up.

Reply:
I will experiment in a separate branch with using a copy and paste of the yaml pulled straight from the API definition.

Reply:
https://github.com/ansys/grantami-bomanalytics/pull/727